host/testcluster.go

```diff
@@ -225,15 +224,6 @@ func newPProfInitializerImpl(logger log.Logger, port int) *pprof.PProfInitialize
 	}
 }
 
-func setupShards(testBase persistencetests.TestBase, numHistoryShards int32, logger log.Logger) {
-	for shardID := int32(1); shardID <= numHistoryShards; shardID++ {
-		err := testBase.CreateShard(int32(shardID), "", 0)
```

Comment:
I was wondering why this was calling CreateShard even though they're created on-demand: turns out this code has been carried around since Jan 2017, before on-demand creation (which came pretty soon after, looks like May 2017).